data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+			`	The three agents get back in their bunks, and say you may depart now.`
+
+
+
+mission "HI2-SF"
+	name "Investigate Stronghold of Flugbu"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Stronghold of Flugbu"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`While you doubt anyone opposing the Heliarchs would find much success in this massive compound, it seems the planet holds enough resources, ships and serves as a valuable location that it was still marked for investigation.`
+			`	The three agents informs you that you're not allowed to enter most of the compound they will be headed to, leaving you behind to stay inside your ship. They return right as the sun is setting, revealing that they've found no signs of the Resistance.`
+
+
+
+mission "HI2-SS"
+	name "Investigate Shifting Sand"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Shifting Sand"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`Immediately upon landing, Iplora is the first to leave your ship, disappearing into the spaceport before you could stop her. You ask Tugroob and Kirrie about it, and they say that she is just taking a small break.`
+			`	"Value moments of solitude, the Saryd do. To a calm oasis, she will head for the day," Kirrie explains, as he also prepares to leave, claiming to know some Heliarchs stationed here who could help him look into some rumors.`
+			`	Being the last Heliarch agent in your ship, Tugroob turns to you, and asks, "Assist me in my search, would you, Captain?"`
+			choice
+				`	"Of course, where are we headed?"`
+					goto desertspider
+				`	"Sorry, I'll have to refuse this time."`
+					goto lonespider
+			label desertspider
+			`	"Reports of suspicious transactions, we have received. An outdated, abandoned warehouse, the place is. For storage of the few crops that can grow here, used, it was," they say as you put in the coordinates they give you.`
+			`	You follow Tugroob to the storage area after landing, spotting a desert storm brewing in the distance as you make your way there. A large crack has formed, around the area where they said the crops were originally planted and tested.`
+			`	The barn itself is quite large and could probably hold more stockpile than one of the medium-sized Arach freighters in it's earlier days, but now many of the planks making up the walls have been torn down in several spots.`
```

Comment:
```suggestion
			`	The barn is large enough that it could have held more stockpile than any medium-sized Arach freighters. Now, many of the planks making up the walls have been torn down in several spots.`
```